weather-map-wk7/client/src/App.js

```diff
@@ -0,0 +1,39 @@
+import './App.css';
+import { useState } from "react";
+import WeatherForm from './components/weatherForm';
+import WeatherCard from './components/weatherCard';
+
+function App() {
+  const [city, setCity] = useState("");
+  const [result, setResult] = useState(null);
+
+  //A function to do the get request and set the state from the hard code data
+  const loadCity = () => {
+    //const params = new URLSearchParams({cityName: city});
+    //const URL = `http://localhost:8080/weather?${params}`;
+    //`http://localhost:8080/api/weather?cityName=${city}`
+    fetch(`http://localhost:8080/weather?cityName=${city}`)
+      .then((response) => response.json())
+      .then((result) => { 
+        //setCity(result);
+        setResult(result);
+        console.log(city);
+        console.log(result);
```

Comment:
On most teams, you are encouraged to remove all `console.log` statements before you deploy an app to a live environment.